### ISSUE TITLE
ValueError: Minimization failed after 100 attempts.

### ISSUE THREAD

**Issue by Kangfengyuuuu** [Zuricho/ParallelFold#46]
Hello, I recently encountered a Value Error: Minimization failed after 100 attempts while using your Parallelfold. I have installed it according to your tutorial and generated the MSA and features.pkl files. May I ask how to resolve this amber error?

**Comment by Zuricho** [comment]
Yes, there are some current issue in the openmm version. You can skip amber relax for now. I'm working on change the environment to solve this problem.

**Comment by Kangfengyuuuu** [comment]
> Yes, there are some current issue in the openmm version. You can skip amber relax for now. I'm working on change the environment to solve this problem.

Thank you for your reply！I ran the entire process on the CPU server. I tried to skip amber relax,  At first, I used the parameter "- r none" and found that it didn't work, but it still reported the same error. Later, I added the parameter "- G" (Disable GPU relax) and there was no error. I checked the output file and found relaxed_model. Does this mean that amber relax can be used again?
